src/main/kotlin/sirius/parent/SiriusParentPlugin.kt

```diff
@@ -107,32 +107,28 @@ class SiriusParentPlugin : Plugin<Project> {
         }
     }
 
-    private fun initializeTestTask(project: Project) {
-        project.tasks.apply {
-            val testTaskFull = getByPath("test") as Test
-            testTaskFull.setIncludes(listOf("**/*TestSuite.class"))
-            testTaskFull.jvmArgs = listOf("-Ddebug=true")
-            testTaskFull.testLogging { logging ->
-                logging.events = setOf(TestLogEvent.FAILED, TestLogEvent.STANDARD_OUT, TestLogEvent.STANDARD_ERROR)
-            }
-            testTaskFull.useJUnitPlatform()
+    private fun TaskContainer.initializeTestTask() {
+        val testTaskFull = getByPath("test") as Test
+        testTaskFull.setIncludes(listOf("**/*TestSuite.class"))
+        testTaskFull.jvmArgs = listOf("-Ddebug=true")
+        testTaskFull.testLogging { logging ->
+            logging.events = setOf(TestLogEvent.FAILED, TestLogEvent.STANDARD_OUT, TestLogEvent.STANDARD_ERROR)
         }
+        testTaskFull.useJUnitPlatform()
     }
 
-    private fun initializeTestTaskWithoutNightly(project: Project) {
-        project.tasks.apply {
-            register("testWithoutNightly", Test::class.java)
-            val testTaskWithoutNightly = getByName("testWithoutNightly") as Test
-            testTaskWithoutNightly.group = "verification"
-            testTaskWithoutNightly.setIncludes(listOf("**/*TestSuite.class"))
-            testTaskWithoutNightly.jvmArgs = listOf("-Ddebug=true")
-            testTaskWithoutNightly.testLogging { logging ->
-                logging.events = setOf(TestLogEvent.FAILED, TestLogEvent.STANDARD_OUT, TestLogEvent.STANDARD_ERROR)
-            }
-            testTaskWithoutNightly.systemProperty("test.excluded.groups", "nightly")
-            testTaskWithoutNightly.useJUnitPlatform { platformOptions ->
-                platformOptions.excludeTags = setOf("nightly")
-            }
+    private fun TaskContainer.initializeTestTaskWithoutNightly() {
```

Comment:
Was macht ein TaskContainer?

Reply:
Der sagt einfach, dass die Methode innerhalb von so einem `project.tasks.apply { ... }` Block aufrufbar ist. Dachte das macht mehr Sinn für die Methoden wo's um Task setup geht.

Gibt theoretisch auch `DependencyContainer`, `PluginContainer`, usw. Aber dort haben wir aktuell nicht genug Setup um das in mehrere Methoden aufzuteilen